CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.0.2
+This is a bugfix release
+
+* Fixed a bug where unrecognized config file extensions caused `Hocon.load` to return an empty
+  hash instead of an error.
+* Added an optional `:syntax` key to the `Hocon.load` method to explicitly specify the file format
+* Renamed internal usage of `name` methods to avoid overriding built in `Object#name` method
```

Comment:
@jpinsonault I think this should probably be a Y release?  Technically that `:syntax` change is a new feature?